src/web/overlays/wwwroot/js/alerts.js

```diff
@@ -73,8 +79,10 @@ const app = new Vue({
 
       if (nextAlert.type === 'onDonation') {
         name = nextAlert.data.user;
-      } else {
+      } else if (nextAlert.data.user) {
         name = nextAlert.data.user.display_name || nextAlert.data.user.login;
+      } else {
```

Comment:
We're not using `name` below so we could probably remove this else and not set the name variable at all.